front/app/components/admin/SurveyDeleteModal/SurveyDeleteModal.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
```

Comment:
Not sure why it registers as a new component, but I just moved this one from an Admin container into the shared components folder :)